src/loaders/swaggerLoader.ts

```diff
@@ -0,0 +1,29 @@
+import { MicroframeworkLoader, MicroframeworkSettings } from 'microframework';
+import * as swaggerUi from 'swagger-ui-express';
+// tslint:disable-next-line:no-var-requires
+const swaggerJSDoc = require('swagger-jsdoc');
```

Comment:
Change this to import

Reply:
@VivekFitkariwala Changed to import statement.